lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -799,9 +797,8 @@ estimateTxSize witnessTag ctx (SelectionSkeleton inputCount outs chgs) =
     sizeOf_Output TxOut {address, tokens}
         = sizeOf_SmallArray
         + sizeOf_Address address
-        + sizeOf_SmallArray
-        + sizeOf_Coin (TokenBundle.getCoin tokens)
-        + F.foldl' (\t -> (t +) . sizeOf_NativeAsset) 0 (TokenBundle.getAssets tokens)
```

Comment:
Sounds good, but does it not risk being slower?

Reply:
> Sounds good, but does it not risk being slower?

I eventually decided to get rid of this. I'm pretty sure that our estimate is quite a bit higher than the real serialized length (for Shelley), but I'd be happier changing this function when we have a test to check that `estimateTxSize` gives answers that are greater than or equal to the results of serializing a transaction with `cardano-node` functions.